hack/boilerplate.go.txt

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
I'm not quite sure if you can delete the whole file. I seems to recall a problem when generating the CRDs... maybe that changed though in recent versions of controller-tools or controller-runtime.

In K8up we had to define it here: https://github.com/vshn/k8up/blob/master/generate.go#L5

Reply:
The file is still there, albeit empty.